src/Service/Modules/ParcelsModule.cs

```diff
@@ -23,37 +23,42 @@ public ParcelsModule(IFacadeService<Parcel, int, ParcelResource, ParcelResource>
 
         private object GetParcel(int id)
         {
-            var results = this.parcelsFacadeService.GetById(id);
+            var result = this.parcelsFacadeService.GetById(id);
+
             return this.Negotiate
-                .WithModel(results)
+                .WithModel(result)
                 .WithMediaRangeModel("text/html", ApplicationSettings.Get)
                 .WithView("Index");
         }
 
         private object GetParcels()
         {
             var resource = this.Bind<SearchRequestResource>();
+
             var results = string.IsNullOrEmpty(resource.SearchTerm)
                               ? this.parcelsFacadeService.GetAll()
                               : this.parcelsFacadeService.Search(resource.SearchTerm);
+
             return this.Negotiate
                 .WithModel(results)
-                .WithMediaRangeModel("text/html", ApplicationSettings.Get)
-                .WithView("Index");
+                .WithMediaRangeModel("text/html", ApplicationSettings.Get);
```

Comment:
I might be totally off here, but is the WithView needed here to serve up the app if going directly to the url in the browser?

Reply:
I think it would be needed but I've got a bit in the constructor that's been cut off by github which I think should do it! The search will be after page load so I think I want the intial pageload call not to bother doing a getAll or search -
` this.Get("/logistics/parcels", _ => this.Negotiate.WithModel(ApplicationSettings.Get()).WithView("Index"));`